docs/backend/fields.md

```diff
@@ -117,11 +120,12 @@ See [`plone.app.textfield`](https://pypi.org/project/plone.app.textfield/) for m
 
 ### Fields in `plone.schema`
 
-See [`plone.schema`](https://pypi.org/project/plone.schema/) for more details.
+See {ref}`backend-ploneschema-label` for more details.
 
 | Name | Type | Description | Properties |
 | ----- | ---- | ----------------------------------- | ------------------ |
-| Email | str | A field containing an email address | IField, IMinMaxLen |
+| Email | str | A field containing an email address  | IField, IMinMaxLen |
+| JSON  |     |                                      |                    |
```

Comment:
the end of the file seems incomplete (table with just the JSON cell).

Reply:
those are just the additional fields in plone.schema. I added the JSON field there, but we need to add description and properties also. But can be done later too.